src/Groups/sub.jl

```diff
@@ -199,6 +292,24 @@ among the nontrivial normal subgroups.
 
 Return the list of characteristic subgroups of `G`,
 i.e., those subgroups that are invariant under all automorphisms of `G`.
+
+# Examples
+```jldoctest
+julia> subgroups(symmetric_group(3))
+6-element Vector{PermGroup}:
+ Permutation group of degree 3 and order 1
+ Permutation group of degree 3 and order 2
+ Permutation group of degree 3 and order 2
+ Permutation group of degree 3 and order 2
+ Permutation group of degree 3 and order 3
+ Permutation group of degree 3 and order 6
+
+julia> characteristic_subgroups(quaternion_group(8))
+3-element Vector{PcGroup}:
+ Pc group of order 1
+ Pc group of order 2
+ Pc group of order 8
```

Comment:
The result of the doctest says that the groups should appear in decreasing order.
I have checked with several GAP versions that this behaviour is reproducible.
How did the claimed result with increasing group orders arise?

Reply:
Apparently the order changes depending on whether the CRIPS package is loaded or not.